multimedia/mpv/Portfile

```diff
@@ -278,9 +278,10 @@ platform darwin {
 
 
 # Shameless copy from ipe-tools.
-set python.versions         {27 36}
-# Remove after 07-30-2018.
-dict set python.legacy_versions 35 36
+set python.versions         {27 37}
+# Remove after 2019-01-16
```

Comment:
I guess you mean 2020-01-16 here?

Reply:
Oops—Yes, I did. Thanks for spotting that

Reply:
Fixed in 3c4bed8bd9c941d7bfec335aa5df4c7243e8f688